src/main/java/step3/ResultNumber.java

```diff
@@ -0,0 +1,42 @@
+package step3;
+
+import com.sun.glass.ui.Clipboard;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ResultNumber {
+    private static List<LottoNumber> resultNumberList = new ArrayList<>();
+    private static LottoNumber bonusNumber;
+
+    public static void settingLottoResultNumber(String resultString) {
+        resultNumberList = new ArrayList<>();
+        for (String number : resultString.split(", ")) {
+            resultNumberList.add(new LottoNumber(Integer.parseInt(number)));
+        }
+    }
+
+    public static void settingLottoBonusNumber(String inputBonusNumber) {
+        bonusNumber = new LottoNumber(Integer.parseInt(inputBonusNumber));
+    }
+
+    public static int checkLottoResult(List<LottoNumber> lottoNumberList) {
+        int winningNumber = 0;
+
+        for (LottoNumber i : resultNumberList) {
+            winningNumber += checkNumberContain(lottoNumberList, i);
+        }
+        return winningNumber;
+    }
+
+    private static int checkNumberContain(List<LottoNumber> lottoNumberList, LottoNumber lottoNumber) {
```

Comment:
간단하게 네이밍해볼 수 있을 것 같아요!
```suggestion
    private static int hasNumber(List<LottoNumber> lottoNumberList, LottoNumber lottoNumber) {
```